flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointITCase.java

```diff
@@ -260,6 +261,10 @@ public UnalignedCheckpointITCase(
                         .setChannelTypes(channelType)
                         .setExpectedFailures(5)
                         .setFailuresAfterSourceFinishes(1)
+                        // prevent test from timing out in case when a failover happens concurrently
+                        // with triggering a checkpoint (some execution status can change right
+                        // after triggering)
+                        .setCheckpointTimeout(Duration.ofMinutes(1))
```

Comment:
Should we set the timeout to a smaller value? 1 minutes sounds still quite long. How long does the UnalignedCheckpointITCase take? Do we have to change any threshold to tolerate checkpoint failures in case they time out?

Reply:
Good point, I'll reduce the timeout and set `TOLERABLE_FAILURE_NUMBER`.
The test usually takes several seconds, up to 20 on my machine.